integration_tests/__tests__/start.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+
+const { runHaul, runHaulSync } = require('../runHaul');
+const { cleanup, writeFiles } = require('../utils');
+const path = require('path');
+const os = require('os');
+const { run } = require('../utils');
+const stripAnsi = require('strip-ansi');
+
+const TEMP_DIR = path.resolve(os.tmpdir(), 'start_test');
+const TEST_PROJECT_DIR = path.resolve(
+  __dirname,
+  '../fixtures/react-native-with-haul',
+);
+
+beforeEach(() => {
+  run('yarn --mutex network', TEST_PROJECT_DIR);
```

Comment:
Why do we want to run `yarn` before each of the suites? `start` doesn't affect `node_modules` at all?